eu.octanne.McBoyard.yml

```
@@ -0,0 +1,64 @@
+app-id: eu.octanne.McBoyard
+separate-locales: false
+
+base: io.atom.electron.BaseApp
+base-version: '19.08'
+
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

this won't work unless the app is working fine under wayland.

---

This app is working fine under wayland.

---

I don't think so, as it requires xrandr.

---

The minecraft repo use --socket=x11 and xrandr
https://github.com/flathub/com.mojang.Minecraft/blob/c4addfc0cc465c5de44ede33aff838312ca3d8e3/com.mojang.Minecraft.json#L28

---

Hence my change request. Fallback-x11 means that the application won't have access to x11 unless wayland isn't available. Instead of having access to both with `--socket=wayland --socket=x11`. Your app can't work with `--socket=wayland` only. So it needs `--socket=x11` instead of `--socket=fallback-x11`

---

ah ok I didn't understand 😁